engine/src/lib.rs

```diff
@@ -13,8 +13,6 @@ extern crate alloc;
 extern crate core;
 
 mod map;
-#[cfg(feature = "meta-call")]
-pub mod meta_parsing;
```

Comment:
Can we delete all of this? We're never going to use it for anything?

Reply:
Weird. I thought that I had caught all of these in my review. Must've not searched the entire library by mistake.

Reply:
We had discussed this before, where we had decided that it shouldn't be the engine that handles meta calls, but it should be the relayer itself, if I'm not mistaken.